### PR TITLE
Added Confetti

### LINKED ISSUE #60: Add a confetti when a user wins/draw happens
Logic should be here - https://github.com/code100x/chess/blob/main/apps/frontend/src/screens/Game.tsx#L55

### PR DESCRIPTION
**Pull request by rahil1202**
Closed #60 
Hey @hkirat if you can review this please ?!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,5 +20,8 @@
   "workspaces": [
     "apps/*",
     "packages/*"
-  ]
+  ],
+  "dependencies": {
```
Comment (hkirat) [inline package.json:24]: Dep should be in the apps/frontend folder

Comment (rahil1202) [reply package.json:24]: Moved to Frontend/package.json

**Comment by nimit9** [comment]
Please add the screenshots incase of UI changes

**Comment by nimit9** [comment]
Please add the screenshots incase of UI changes

**Pull request closed without merging**